fili-core/src/main/java/com/yahoo/bard/webservice/web/FiliResponseWriter.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+
+/**
+ * Default writer for response. FiliResponseWriter choose a proper writer type using FiliResponseWriterSelector.
+ */
+public class FiliResponseWriter implements ResponseWriter {
+
+    private final FiliResponseWriterSelector responseWriterSelector;
+
+    /**
+     * Constructor.
+     *
+     * @param responseWriterSelector  Used to decided writer type when called write.
+     */
+    public FiliResponseWriter(FiliResponseWriterSelector responseWriterSelector) {
```

Comment:
@archolewa This is a very educative explanation! Stupid me, can't believe I thought `com.java.astrology` exist for a few seconds.